code/modules/maps/triumph/levels/lavaland.dm

```diff
@@ -57,3 +57,29 @@ LAVALAND_TURF_CREATE(/turf/simulated/floor/bluegrid)
 LAVALAND_TURF_CREATE(/turf/simulated/floor/greengrid)
 LAVALAND_TURF_CREATE(/turf/unsimulated/mineral/triumph)
 LAVALAND_TURF_CREATE(/turf/simulated/mineral/)
+
+
+/turf/simulated/mineral/floor/lavaland
+	outdoors = TRUE
+
+//Cave doesnt make things indoor, its actually a variant that
+//has an initial gas string of regular air mix. The turf creator overrides
+//that gas mix though so this is effectively useless. Lava land was made though with this
+//being default outdoors so I will leave it as is for now
+/turf/simulated/mineral/floor/cave/lavaland
+	outdoors = TRUE
+
```

Comment:
This file is depreciated. It will get deleted when we organize turfs, I'm only changing it right now as to keep it consistent with the other files changes